### PR TITLE
MIlestone 1: Header and Hero sections

### PR DESCRIPTION
**Pull request by anagudelogu**
## Portfolio milestone 1:
In this milestone, I:
- Set up HTML and CSS linters for the project.
- Added HTML and Styles for the mobile version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <img src="./images/ic_menu.svg" alt="Menu" title="Menu">
+    </header>
```
Comment (mahbubul14) [inline index.html:13]: - [ ] The hamburger button on the right should include in a `nav` tag. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @anagudelogu ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Currently, your `README` file is missing with the "Getting Started"  section. Please work on that.

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (anagudelogu) [reply index.html:13]: Hi @mahbubul14! 

Do I need to add a "Getting started section" if, at this moment, the project doesn't need any particular prerequisites, set up, install, usage, tests, or deployment?

**Review by gemmen29 — APPROVED** [review]
Hi @anagudelogu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- Kindly give the branch a meaningful name like the name of the project `setup-and-mobile-version-skeleton` not `section-1-2`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by anagudelogu** [comment]
Thank you! @gemmen29

File: index.html (hunk 1 above)
Comment (mahbubul14) [reply index.html]: Hello @anagudelogu,
I can see your project is already approved. So, you don't need to add anything more. 
You can check [this link](https://github.com/mahbubul14/house_rent) to get an idea for the near future. Those parts you haven't worked on like "test", you don't need to add. You can just add a basic idea of how people can run the project.

**Pull request merged**